site/content/blog/2022-06-01-whats-new-in-svelte-june-2022.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "What's new in Svelte: June 2022"
+description: "Cancellable dispatched events, deeper {@const} declarations and more!"
+author: Daniel Sandoval
+authorURL: https://desandoval.net
+---
+
+With last month's [Svelte Summit](https://www.youtube.com/watch?v=qqj2cBockqE) behind us, we're ready to apply everything we learned in this new month of June! Also new this month are some quality-of-life changes to `createEventDispatcher`, `@const` declarations and tons of progress toward SvelteKit 1.0.
+
+Let's dive in!
+
+## What's new in Svelte
+- Custom events can now be cancelled in the `createEventDispatcher` function (**3.48.0**, [Docs](https://svelte.dev/docs#run-time-svelte-createeventdispatcher), [PR](https://github.com/sveltejs/svelte/pull/7064))
+- The `{@const}` tag can now be used in `{#if}` blocks to conditionally define variables (**3.48.0**, [Docs](https://svelte.dev/docs#template-syntax-const), [PR](https://github.com/sveltejs/svelte/pull/7451))
+- Lots of bug fixes across `<svelte:element>`, animations and various DOM elements. Check out the [CHANGELOG](https://github.com/sveltejs/svelte/blob/master/CHANGELOG.md#3480) for a deeper dive!
+
+
+## What's new in SvelteKit
+- Vite 2.9.9 was released as one of the last Vite 2 releases. The Svelte team has been hard at work contributing to the the Vite 3 release to make the integration between SvelteKit and Vite smoother than ever ([Vite 3.0 Milestone](https://github.com/vitejs/vite/milestone/5))
+- `config.kit.alias` lets you more easily declare a custom alias to replace values in `import` statements ([Docs](https://kit.svelte.dev/docs/configuration#alias), [Issue](https://github.com/sveltejs/kit/pull/4922))
```

Comment:
i know this is already merged but the issue link here is incorrect, it is the pr for removing node 14 support. it should be `pull/4964` or `issue/4734`.

Reply:
i'll fix it. thanks

Reply:
Good catch. Thank you!